distributions/openhab/src/main/resources/bin/update

```diff
@@ -62,7 +62,7 @@ setup(){
   fi
 
   ## Check to see if processes are running before updating
-  if [ ! -z "$(ps aux  | grep "openhab2.*java" | grep -v grep)" ]; then
+  if [ ! -z "$(ps aux  | grep openhab | grep -v grep)" ]; then
```

Comment:
@wborn I need your expertise, please. This change didn't have seemed to work out - the update script now always claims that openHAB is running, even if it isn't. I done really understand what was the idea behind `"openhab2.*java"` - can you help to fix this?

Reply:
There is probably another process running that has "openhab" in its path or arguments. If I run Eclipse it also matches "openhab". So "openhab2.*java" is to prevent false positives and only match the real openHAB java process.

Reply:
Checking it, I see that you are right - there is also a shell process for `runtime/bin/karaf`, which happens to run inside an `openhab` folder...